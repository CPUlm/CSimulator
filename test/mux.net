INPUT a,b,c
OUTPUT d
VAR
  a , b, c, d
IN
d = MUX a b c
