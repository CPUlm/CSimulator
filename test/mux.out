Step 1: 
=> d = 0
Step 2: 
=> d = 1
Step 3: 
=> d = 0
Step 4: 
=> d = 1
